python/docs/source/reference/pyspark.sql/functions.rst

```diff
@@ -64,31 +64,39 @@ Math Functions
     bin
     cbrt
     ceil
+    ceiling
     conv
     cos
     cosh
     cot
     csc
+    e
     exp
     expm1
     factorial
     floor
     hex
     unhex
     hypot
+    ln
     log
     log10
     log1p
     log2
+    negative
```

Comment:
@HyukjinKwon they were added here. except the `negate`, I will add it.

BTW, in the last commit I add `negate` (which only existed in scala) to the python side for the parity between py and scala